### DISCUSSION TITLE
Discord bot

### DISCUSSION THREAD

**Discussion by veiyooz** [Rapptz/discord.py#9407] [General]
i made a discord bot but i want to code autoban if someone joins my server they get 2 sec timer then gets banned

**Comment by almightynan** [comment]
no one here will write code for you

**Comment by No767** [comment]
Just dont

**Comment by hsprakash** [comment]
you can watch a youtube vid on how make a ban command and just check upon member join if 2 seconds has passed, if yes ban user.
